src/utils/daos_autotest.c

```diff
@@ -186,6 +222,9 @@ oS1(void)
 	new_oid();
 	daos_obj_generate_oid(coh, &oid, 0, OC_S1, 0, 0);
 
+	total_nr = 1000000;
+	setup_progress();
+
 	for (i = 0; i < 1000000; i++) {
```

Comment:
should use "i < total_nr" now, right ?

Reply:
That's a good point. I did consider that but I was hesitant using total_nr here as that variable use is for the progress bar only. I also considered creating a new variable for the number of records. That way total_nr and the for loop would use that variable. If I had to choose the former option would be the cleanest. I can proceed with that.